apple/DemoApp/Demo/DemoNavigationView.swift

```diff
@@ -35,6 +32,10 @@ struct DemoNavigationView: View {
     @State private var camera: MapViewCamera = .center(initialLocation.coordinate, zoom: 14)
     @State private var snappedCamera = true
 
+    // NOTE: This is probably not ideal but works for demo purposes.
+    // This causes a thread performance checker warning log.
+    @StateObject private var spokenInstructionObserver = AVSpeechSpokenInstructionObserver(isMuted: false)
```

Comment:
Maybe this does not actually have to be a StateObject? Maybe it can be an `ObservedObject` instead like `FerrostarCore` is.